db/migrate/20151216095054_create_container_builds.rb

```diff
@@ -0,0 +1,22 @@
+class CreateContainerBuilds < ActiveRecord::Migration
+  def change
+    create_table :container_builds do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.timestamp  :creation_timestamp
+      t.string     :resource_version
+      t.string     :namespace
+      t.string     :service_account
+      t.string     :build_source_type
+      t.string     :binary
```

Comment:
@enoodle reading in the doc this seems to be a tar file (I could be wrong, please double check). If that's the case then it would be overkill to persist it (don't collect it).